code/game/machinery/computer/communications.dm

```diff
@@ -221,7 +227,7 @@ var/list/shuttle_log = list()
 			if(!ert_reason)
 				to_chat(usr, "<span class='warning'>You are required to give a reason to call an ERT.</span>")
 				return
-			if(!Adjacent(usr) || usr.incapacitated())
+			if((!usr.Adjacent(src) && !issilicon(usr)) || usr.incapacitated())
```

Comment:
Undocumented change and I'm skeptical of giving Malf AIs the ability to call ERTs. We have already had issues in the past related to traitors calling ERTs leading to an administrative announcement on this subject. Since the station is only permitted to call an ERT one time this seems bad.

Reply:
You're right I'll remove this.